SU-SWS/stanford_jumpstart_home
# HSDO-748 Added checkbox on customize display page

# READY FOR REVIEW

# Summary
- Allows the addition of an optional css file that mostly changes font sizes.

# Needed By (Date)
- 3/15

# Steps to Test

1. Checkout branch.
2. run database updates
3. view /admin/stanford-jumpstart/customize-design and confirm "Include BIG text " checkbox near bottom
4. check the box and save
5. confirm font sizes change.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSDO-753
- https://github.com/SU-SWS/linky_clicky/pull/145
